src/main/java/com/luxoft/svinenkov/movieland/dao/jbbc/MovieDaoImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.luxoft.svinenkov.movieland.dao.jbbc;
+
+import com.luxoft.svinenkov.movieland.dao.MovieDao;
+import com.luxoft.svinenkov.movieland.dao.jbbc.mapper.MovieRowMapper;
+import com.luxoft.svinenkov.movieland.entity.Movie;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class MovieDaoImpl implements MovieDao {
+    private final Logger log = LoggerFactory.getLogger(getClass());
+    private static final MovieRowMapper movieRowMapper = new MovieRowMapper();
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @Autowired
+    private NamedParameterJdbcTemplate namedJdbcTemplate;
```

Comment:
not used

Reply:
"private NamedParameterJdbcTemplate namedJdbcTemplate;" removed